CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
    - `native_bounding_box` with coordinates and CRS of native CRS  [minx, miny, maxx, maxy, "EPSG:&lt;code&gt;"]
 - [#64](https://github.com/LayerManager/layman/issues/64) Layer thumbnails are generated in native CRS of the layer.
 - [#489](https://github.com/LayerManager/layman/issues/489) Error responses from Micka and GeoServer are logged into log and also propagated as part of raised exception, so they can be seen from flower.
+- [#548](https://github.com/LayerManager/layman/pull/548) Handle GeoServer HTTP error 409 when setting layer access rights.
```

Comment:
Supress GeoServer HTTP error 409 when setting layer access rights, if they are set to the same value.